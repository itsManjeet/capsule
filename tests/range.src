#![EXPERIMENTAL_FEATURES]
for i in range (5) {
    println (i);
}

for i in range (5) {
    if i == 2 {
        continue;
    }
    println (i);
}

for i in range(5) {
    for j in range(5) {
        println (i, j);
    }
}

for i in range(5) {
    if i == 3 {
        continue;
    }
    for j in range(5) {
        if j == 2 {
            continue;
        }
        println (i, j);
    }
}