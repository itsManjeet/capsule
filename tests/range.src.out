0
1
2
3
4
0
1
3
4
00
01
02
03
04
10
11
12
13
14
20
21
22
23
24
30
31
32
33
34
40
41
42
43
44
00
01
03
04
10
11
13
14
20
21
23
24
40
41
43
44
